### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by isaac-chung** [embeddings-benchmark/mteb#1269] [General]
Are we open to opt in for https://hacktoberfest.com/? All we need is to add a label, pick some issues from the backlog, and spell out the details a bit more.

(After the ICLR submission, of course)

CC @KennethEnevoldsen @x-tabdeveloping @imenelydiaker @Muennighoff @Samoed and anyone who might be interested/ helping with reviews already.

**Comment by KennethEnevoldsen** [comment]
We could def. do that. We have a few datasets additions that are reasonable. Though I don't have too much time in Oktober (assuming that I will be working more on MIEB)

**Reply by isaac-chung** [reply]
I could add the label to some of the "Good First Issues" ones.

**Comment by isaac-chung** [comment]
Added [this issue](https://github.com/embeddings-benchmark/mteb/issues/1279) and the "hacktoberfest" label.

**Reply by KennethEnevoldsen** [reply]
Thanks @isaac-chung!
